src/server.rs

```diff
@@ -60,3 +76,31 @@ pub fn handle_connection(
         }
     }
 }
+
+pub fn new_p2p_connection(
+    conn_id: u64,
+    peer_address: String,
+    peer_port: u16,
+    messenger: Sender<MessengerOperations>,
+    receiver: Receiver<P2POperations>,
+) -> Uuid {
+    let peer_info = format!("{}:{}", peer_address, peer_port.to_string());
+    let stream = TcpStream::connect(peer_info).unwrap();
+
+    messenger

```

Comment:
This is also probably doesn't belong here- the fact that this connection is relevant to the messenger is beyond the scope of the server. The server module should just understand connections, how to make them and how to listen to them.